packages/core/src/config.js

```diff
@@ -13,7 +13,7 @@ export const DEFAULT_CONFIG = {
   replaceAttrValues: null,
   svgProps: null,
   svgo: true,
-  svgoConfig: null,
+  svgoConfig: { plugins: [{ name: 'convertStyleToAttrs', active: true }]},
```

Comment:
This plugin is error prone and was disabled because of it. I don't recommend to enable it.